keyboards/handwired/lumawing/keyboard.json

```
@@ -0,0 +1,95 @@
+{
+    "keyboard_name": "LuMaWing",
+    "manufacturer": "Lucas Mateijsen",
+    "url": "",
+    "maintainer": "qmk",
+    "usb": {
+        "vid": "0xBB80",
+        "pid": "0x050D",
+        "device_version": "0.0.1"
+    },
+    "features": {
+        "bootmagic": false,
+        "command": true,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": false
+    },
+    "qmk": {
+        "locking": {
+            "enabled": true,
+            "resync": true
+        }
+    },
+    "matrix_pins": {
+        "cols": ["B1", "B3", "B2", "B6", "B5", "B4", "E6", "D7", "C6", "D4", "D0", "D1", "D2", "D3"],
+        "rows": ["F4", "F5", "F6", "F7"]
+    },
+    "diode_direction": "COL2ROW",
+    "processor": "atmega32u4",
+    "bootloader": "caterina",
```

```suggestion
    "development_board": "promicro",
```
This sets this, and pin compatibility, which is used for converters.

---

Applied your suggestion and removed the processor and bootloader line and replaced it with development_board.